exchanges/graphcache/src/operations/write.ts

```diff
@@ -239,6 +239,7 @@ const writeSelection = (
     typename,
     entityKey || typename,
     deferRef,
+    false,
```

Comment:
```suggestion
    optionalRef,
```

I might be wrong, but this could accidentally cascade to this being reset to `false`, right?

This is a mistake on my part because, ultimately, `makeSelectionIterator` shouldn't accept either defer or optional as arguments, but that was carried over from past buggy behaviour.
It's just inconvenient for testing